frontend/packages/dev-console/src/components/add/DeveloperFeaturesGettingStartedCard.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FlagIcon } from '@patternfly/react-icons';
+
+import { ALL_NAMESPACES_KEY, useActiveNamespace, useOpenshiftVersion } from '@console/shared/src';
+import { GettingStartedCard } from '@console/shared/src/components/getting-started';
+import { GettingStartedLink } from '@console/shared/src/components/getting-started/types';
+
+export const DeveloperFeaturesGettingStartedCard = () => {
+  const { t } = useTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  // Show only major and minor version.
+  const version = (useOpenshiftVersion() || '')
+    .split('.')
+    .slice(0, 2)
+    .join('.');
+
+  const links: GettingStartedLink[] = [
+    {
+      key: 'helm-charts',
+      title: t('devconsole~Discover certified Helm Charts'),
+      href:
+        activeNamespace && activeNamespace !== ALL_NAMESPACES_KEY
+          ? `/catalog/ns/${activeNamespace}?catalogType=HelmChart`
+          : '/catalog/all-namespaces?catalogType=HelmChart',
```

Comment:
I'm not sure how `ALL_NAMESPACES_KEY` would work here. we show the project list page when all namespace is selected on add page.
This getting started card will only show up when we have an active namespace. I dont think you need to create all-namepaces url.

Reply:
There is a small edge case: When the user has no access to any namespace, opens the topology, she / he still see the add page. It might be broken also in some other cases.

I will create an issue for that and will ask PM if we need this "feature". If we are sure that we have always a namespace defined, I'm with you that this condition is not required here.